CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.30.0 Release
+- Release Date: April 16, 2020
+- Server Versions Supported: Server v5.19+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
+**Note:** v5.9.0 as our Extended Support Release (ESR) is coming to the end of its lifecycle. v5.19.0 will continue to be our current existing ESR until October 15, 2020. [Learn more in our forum post](https://forum.mattermost.org/t/upcoming-extended-support-release-updates/8526).
+
+**Note:** The Channel Moderation Settings feature is supported on mobile app versions v1.30 and later. In earlier versions of the mobile app, users who attempt to post or react to posts without proper permissions will see an error.
+ 
+### Improvements
+ - Improved Mobile performance related to themes and links.
+ - Added support for different interactive message button styles on mobile.
+ - Enter key on hardware Android keyboard now posts a message.
+ - Added "Unarchive Channel" option to the channel info screen.
+ - Removed mark as unread option from post menus for archived channels.
+ 
+### Bug Fixes
```

Comment:
* Fix an issue on Android when downloading a file or video was not reporting progress
* Fix an issue on Android that was preventing to share content through the share extension
* Fix an issue where Markdown tables was rendering in full in the center channel on larger screen sizes
* Make the name displayed consistent with teammate display name setting
* Fix some selected emojis in autocomplete from rendering properly when posted